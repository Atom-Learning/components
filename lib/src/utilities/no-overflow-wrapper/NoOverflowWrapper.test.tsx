import { render } from '@testing-library/react'
import * as React from 'react'

import { NoOverflowWrapper } from '.'

describe(`NoOverflowWrapper component`, () => {
  it('renders', async () => {
    const { container } = render(<NoOverflowWrapper />)
    expect(container).toMatchSnapshot()
  })
})
