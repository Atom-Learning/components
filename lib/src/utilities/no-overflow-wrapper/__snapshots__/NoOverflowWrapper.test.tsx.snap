// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoOverflowWrapper component renders 1`] = `
@media  {
  .c-edXhKp {
    height: 100%;
    width: 100%;
    border-radius: inherit;
    overflow: hidden;
  }
}

<div>
  <div
    class="c-PJLV c-edXhKp"
  />
</div>
`;
